src/core/thread/network_data.hpp

```diff
@@ -537,7 +537,6 @@ class NetworkData : public InstanceLocator
     };
 
     const Type mType;
-    bool       mLastAttemptWait;
```

Comment:
Is `mLastAttemptWait` useful when the node is just started and current time is within `kDataResubmitDelay` (300s) ?

Reply:
`mLastAttempt` with value 0 would be equivalent to  `mLastAttemptWait` with value`false`.

See the former condition in https://github.com/openthread/openthread/pull/4485/files#diff-c3f068e31220c794c63fd26918c84a85R981.